fontforge/encoding.c

```diff
@@ -622,20 +622,29 @@ static Encoding *ParseGlyphOrderAndAliasDB(FILE *file) {
     }
 
     if (max != -1) {
+        // If we have mappings, we make an encoding.
+        // We pad the map in accordance with existing code in FindOrMakeEncoding and elsewhere.
+        // Nobody knows why.
         if (++max < 256) max = 256;
-        item = calloc(1,sizeof(Encoding));
-        char* buf = strdup(_("Please name this encoding"));
-        item->enc_name = ff_ask_string(buf, "GlyphOrderAndAliasDB", buf);
-        item->char_cnt = max;
-        item->unicode = malloc(max*sizeof(int32));
-        memcpy(item->unicode, encs, max*sizeof(int32));
-
-        if (any) {
-            item->psnames = calloc(max, sizeof(char *));
-            memcpy(item->psnames, names, max*sizeof(char *));
+        char *tmp_name = ff_ask_string(_("Encoding name."), "GlyphOrderAndAliasDB", _("Please name this encoding."));
```

Comment:
I see elsewhere in `encoding.c` that there's a string: `Please name this encoding` (without the period at the end) - I think we should match it with this so that it doesn't mess with the translation strings/ so that we don't have to translate another string